.jshintrc

```diff
@@ -1,4 +1,7 @@
 {
+  "sub": true,
+  "multistr": true,
+  "-W018": true,
```

Comment:
Sorry, I was trying to say that we should fix the issues where "confusing use of '!'" gave a warning, not exclude the rule.

Reply:
Ok i will look into it, but I'm not sure why its showing a warning